docs/kratos/social-signin/40_auth0.mdx

```diff
@@ -4,8 +4,76 @@ title: Add Auth0 as a social sign-in provider in Ory
 sidebar_label: Auth0
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # Auth0
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add Auth0 as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the Auth0 logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create an Auth0 Application for your Auth0 Tenant](https://auth0.com/docs/applications), noting the following points:
+    - Choose the application type to be **Regular Web Applications**.
+    - Paste the saved **Redirect URI** from Ory into the **Allowed Callback URLs** field in the **Settings** tab of the registered application.
+5. Navigate to the **Settings** tab of the registered application on the Auth0 dashboard and copy the following data
+   to the corresponding fields in the form on the Ory Console:
+    - **Client ID**
+    - **Client Secret**
```

Comment:
After completing the previous step I'm already where I should be, right? No need to "navigate". Also, simpler and less fancy verbs ftw - "Go", "Open"

Reply:
Changed to "Go".